tests/unit/test-class-wc-payment-gateway-wcpay-subscriptions-process-payment.php

```diff
@@ -282,10 +282,9 @@ public function test_saved_card_subscription() {
 		$subscriptions = [ WC_Helper_Order::create_order( self::USER_ID ) ];
 		$this->mock_wcs_get_subscriptions_for_order( $subscriptions );
 
-		$this->mock_api_client
-			->expects( $this->once() )
-			->method( 'create_and_confirm_intention' )
-			->with( $this->anything(), $this->anything(), self::PAYMENT_METHOD_ID, self::CUSTOMER_ID, $this->anything(), false, false, $this->anything(), $this->anything(), false )
+		$request = $this->mock_wcpay_request( Create_And_Confirm_Intention::class );
```

Comment:
Missing parameters here as well. Notice how the 7th parameter (`$save_payment_method_to_store`) is not the same as the previous test.

Reply:
Setters for filling this data have been added to test that will cover this case

Reply:
I am sorry. The linter prevented commit, and I didn't notice it until now. It is pushed.

Reply:
Are you sure?
<img width="1526" alt="image" src="https://user-images.githubusercontent.com/5311119/205291460-d6b585f0-813c-4f9c-96ff-70f957471e44.png">

Reply:
The new asseerts have been added.